source/geometry/FloatPoint2D.ooc

```diff
@@ -21,8 +21,10 @@ use ooc-base
 
 FloatPoint2D: cover {
 	x, y: Float
```

Comment:
You mean `FloatTransform3D`? I see your point. I guess this is one of those judgement calls you have to make when writing the class/cover.